src/gui/MainWindow.cpp

```diff
@@ -245,6 +246,7 @@ MainWindow::MainWindow()
     m_ui->actionEntryDelete->setShortcutVisibleInContextMenu(true);
     m_ui->actionEntryClone->setShortcutVisibleInContextMenu(true);
     m_ui->actionEntryTotp->setShortcutVisibleInContextMenu(true);
+    m_ui->actionEntryDownloadIcon->setShortcutVisibleInContextMenu(true);
```

Comment:
Not really a bug, but would be nice to enable the feature when more than one entry is selected.